### PR TITLE
chore: run npm-audit to address alert on the ansi-regex dependency

### PR DESCRIPTION
**Pull request by shivlaks**
mocha@7.1.1 is transitively bringing in various versions of ansi-regex
in the version range (> 2.1.1, < 5.0.1) which has a Moderate severity
security risk and is vulnerable to CVE-2021-3807

also bumped minor version from `1.7.0` -> `1.7.1` so we release to npm

see: GitHub Advisory Database - https://github.com/advisories/GHSA-93q8-gq69-wqmw

---

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by vishalsatam-aws — APPROVED** [review]
Thanks for the quick fix.

File: package.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
 		"@types/mocha": "^5.2.7",
 		"@types/node": "^12.11.7",
 		"json-schema-ref-parser": "^7.1.3",
-		"mocha": "^7.1.1",
```
Comment (wong-a) [inline package.json:33]: We also need to update the `version` field in package.json to release a new version of this package to npm

**Review by ca-nguyen — APPROVED** [review]

**Review by vishalsatam-aws — DISMISSED** [review]

File: package-lock.json [hunk 2]
```diff
@@ -5,6 +5,7 @@
 	"requires": true,
 	"packages": {
 		"": {
+			"name": "amazon-states-language-service",
 			"version": "1.7.0",
```
Comment (wong-a) [inline package-lock.json:9]: package-log.json's version is still 1.7.0. `npm version patch` should update both files

Comment (shivlaks) [reply package-lock.json]: i should* have run yarn test, but reverted and ran `npm version patch` to let the tooling do it's thing.
good catch.

**Review by wong-a — APPROVED** [review]

**Pull request merged**